governance/roles.md

```diff
@@ -76,6 +76,8 @@ navigate a complex security landscape. If the TAG has less than two Technical
 Leads, any Chair may act as Technical Lead.
```

Comment:
The TAG has grown past the point where "If the TAG has less than two Technical Leads, any Chair may act as Technical Lead." can be removed. There is the text that supersedes this with "Chairs may additionally perform any actions of technical leads as needed, especially serving as the Security TAG leadership representative to a project."